nuxeo-java-client-test/src/test/java/org/nuxeo/client/test/TestCurrentUser.java

```diff
@@ -93,4 +96,23 @@ public void itCanChangeAuthMethod() {
         assertEquals("user1", currentUser.getUsername());
     }
 
+    @Test
+    public void itCanLoginWithLongCredentials() {
+        login();
+        String email = "verylongmailaddress0123456789@nuxeo.com";
+        String password = "verylongpassword0123456789";
```

Comment:
It would be better if you can assert that the generated token is greater than 76 characters.